README.md

```diff
@@ -44,25 +45,186 @@ cd cdp-uploader
 nvm use
 ```
 
-## Local development
+# API
 
-### Setup
+## POST /initiate
+
+Example `/initiate` request:
+
+```json
+{
+  "redirect": "https://myservice.com/nextPage",
+  "scanResultCallbackUrl": "https://myservice.com/callback",
+  "destinationBucket": "myservice",
+  "destinationPath": "scanned",
+  "metadata": {
+    "customerId": "1234",
+    "accountId": "1234"
+  }
+}
+```
+
+#### HTTP Headers:
+
+| Header name | Description                                       | Required |
+| ----------- | ------------------------------------------------- | -------- |
+| User-Agent  | Identifier of the service that calls cdp-uploader | TBD      |
+
+#### Body parameters:
+
+| Parameter name        | Description                                                        | Required |
+| --------------------- | ------------------------------------------------------------------ | -------- |
+| redirect              | Url to redirect to after file has been successfully uploaded.      | yes      |
+| destinationBucket     | S3 bucket that file will be moved to once the scanning is complete | yes      |
+| destinationPath       | 'Folder' in bucket where scanned files will be placed              | no       |
+| scanResultCallbackUrl | Url that will be called once all files in upload have been scanned | no       |
+| metadata              | Map of additional information related to upload                    | no       |
```

Comment:
Will be added to the S3 file?

Reply:
Need to double check that...

Reply:
Add to note perhaps

Reply:
We don't add to S3 metadata

Reply:
Ah no we dont, I thought we did. Nevermind.